samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/GradingService.java

```diff
@@ -1567,11 +1567,14 @@ public double getAnswerScore(ItemGradingData data, Map publishedAnswerHash)
     {
     	// return (double) 0;
     	// Para que descuente (For discount)
+    	double score;
```

Comment:
can just set the score to 0 here and then remove the else statement

Reply:
@ottenhoff Just updated!